src/datasources/users/entities/__tests__/users.entity.db.builder.ts

```diff
@@ -0,0 +1,14 @@
+import type { IBuilder } from '@/__tests__/builder';
+import { Builder } from '@/__tests__/builder';
+import { faker } from '@faker-js/faker/.';
+import type { UUID } from 'crypto';
+import type { User } from '@/datasources/users/entities/users.entity.db';
+import { UserStatus } from '@/domain/users/entities/user.entity';
+
+export function userBuilder(): IBuilder<User> {
+  return new Builder<User>()
+    .with('id', faker.string.uuid() as UUID)
+    .with('status', faker.helpers.enumValue(UserStatus))
+    .with('created_at', new Date())
+    .with('updated_at', new Date());
```

Comment:
Nit: we could use faker to have different dates here: `created_at` in the past and `updated_at` thereafter.

Reply:
Lets do it once we start writing tests with this builder